src/components/IconButton/IconButton.tsx

```diff
@@ -47,7 +51,13 @@ export const IconButton = React.forwardRef<HTMLButtonElement, IconButtonProps>(f
   },
   ref
 ) {
-  const styles = useIconButtonStyles({ variantColor, variant, variantBorderStyle, size });
+  const styles = useIconButtonStyles({
+    variantColor,
+    variantIconColor,
```

Comment:
@3nvi @eliskaachee Perhaps we should name this `iconColor` since the icon color itself may be independent of the bg color of the alert

thoughts?

In a perfect world I could imagine having the `variant` prop being `success | info | error | warn | default` and all colors derive from that. Or the "color" prop being a more minimal value than every color in the theme.

But that would be a much larger change, given we already have a `variant` prop and we currently do color as `variantColor`. I'd personally advocate we name the prop `iconColor`

Reply:
I'm indifferent here. I'd say both make sense, since we already have `variantColor`